src/actions/__tests__/historyActions.test.js

```diff
@@ -157,14 +160,15 @@ const transformedImportedPlrTransaction = buildHistoryTransaction({
 describe('History Actions', () => {
   const transactionsHistoryStep = 10;
 
-  const api = {
+  // $FlowFixMe
+  const api: SDKWrapper = {
```

Comment:
Not sure, but maybe we can use `$Shape<SDKWrapper>` to avoid the `$FlowFixMe` if it's the case?

Reply:
Good catch 👍